### PR TITLE
Turn all http:// links to https://

### PR DESCRIPTION
**Pull request by hartwork**

### CODE REVIEW

File: tests/acutest.h [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  * Acutest -- Another C/C++ Unit Test facility
- * <http://github.com/mity/acutest>
+ * <https://github.com/mity/acutest>
```
Comment (mity) [inline tests/acutest.h]: This one would eventually get lost the next time when we update acutest.h from upstream.

The right way is as follows:
1. Get this change to upstream (yep. follow that link ;-);
2. And then we can update the whole header from there.

**Review by mity — COMMENTED** [review]
Looks good to me, except the change in `acutest.h`.

File: tests/acutest.h (hunk 1 above)
Comment (hartwork) [reply tests/acutest.h]: I see. I have opened https://github.com/mity/acutest/pull/43 now and dropped the change to `tests/acutest.h` from this pull request right here, now.

Comment (mity) [reply tests/acutest.h]: Ok, thanks. I'll merge as it is. There are other changes in acutest.h so I'll update that separately outisde the PR.

**Comment by mity** [comment]
Merged. Thanks.

**Pull request merged**